installer/install.iss

```diff
@@ -102,7 +102,7 @@ Filename: {app}\ReleaseNotes.html; Description: View Release Notes; Flags: shell
 [Files]
```

Comment:
I think it would make more sense to squash this into the commit refactoring the code into the new `render-release-notes.sh` script.